### PR TITLE
feat(registration-pages): add sponsorship ticket introduction and set…

### PR DESCRIPTION
**Pull request by tyuchx**
## Types of changes

* **New feature**

## Description

Add sponsorship ticket introduction and set sponsorship ticket open

## Additional context

CleanShot 2022-07-12 at 04 12 25

CleanShot 2022-07-12 at 04 13 00

### CODE REVIEW

**Comment by tyuchx** [comment]
@mattwang44 @josix @uranusjr 
please help me confirm this pr

File: i18n/registration/tickets.i18n.js [hunk 1]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
             'with our Super Early Bird prices. ' +
             'The fee you spend will be put straight back into the community to ' +
             'support our diversity initiatives and financial aid programs. ' +
-            'If you need financial aid, please visit {fa}.{br}' +
-            'Limited edition conference swags Pyckage will be revealed soon.',
+            'If you need financial aid, please visit {fa}.{br}',
+        introSponsor: 'About SponsorShip Pass',
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:23]:
```suggestion
        introSponsor: 'About Sponsorship Pass',
```

File: i18n/registration/tickets.i18n.js [hunk 2]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
             'with our Super Early Bird prices. ' +
             'The fee you spend will be put straight back into the community to ' +
             'support our diversity initiatives and financial aid programs. ' +
-            'If you need financial aid, please visit {fa}.{br}' +
-            'Limited edition conference swags Pyckage will be revealed soon.',
+            'If you need financial aid, please visit {fa}.{br}',
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:22]:
```suggestion
            'If you need financial aid, please visit {fa}.',
```

File: i18n/registration/tickets.i18n.js [hunk 3]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
...
+        introSponsor: 'About SponsorShip Pass',
+        SponsorContent:
+            'PyCon TW events including PyCon APAC 2022 main conference, Sprints, Job Fair, Open Space, PyNight, ' +
+            'PyCast, {br} local Python community events, as well as Young inspirers.' +
+            'and local Python community events.{br} ' +
+            'The last one is fairly new for this year’s participants, where students can share their unique Python learning experiences. {br}' +
+            'You are the motivation for us organizing more PyCon TW events and encouraging ' +
+            'PyCon TW to expand Python possibilities. {br}',
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:30]:
```suggestion
            'PyCon TW to expand Python possibilities.',
```

File: i18n/registration/tickets.i18n.js [hunk 4]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
...
             'support our diversity initiatives and financial aid programs. ' +
-            'If you need financial aid, please visit {fa}.{br}' +
-            'Limited edition conference swags Pyckage will be revealed soon.',
+            'If you need financial aid, please visit {fa}.{br}',
+        introSponsor: 'About SponsorShip Pass',
+        SponsorContent:
+            'PyCon TW events including PyCon APAC 2022 main conference, Sprints, Job Fair, Open Space, PyNight, ' +
+            'PyCast, {br} local Python community events, as well as Young inspirers.' +
+            'and local Python community events.{br} ' +
+            'The last one is fairly new for this year’s participants, where students can share their unique Python learning experiences. {br}' +
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:28]:
```suggestion
            'PyCast, {br} and various local Python community events.{br} ' +
            'We also invited youthful Pythonistas to our Young Inspirers special online event to share their unique Python learning experiences. {br}' +
```

File: i18n/registration/tickets.i18n.js [hunk 5]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
             'with our Super Early Bird prices. ' +
             'The fee you spend will be put straight back into the community to ' +
             'support our diversity initiatives and financial aid programs. ' +
-            'If you need financial aid, please visit {fa}.{br}' +
-            'Limited edition conference swags Pyckage will be revealed soon.',
+            'If you need financial aid, please visit {fa}.{br}',
+        introSponsor: 'About SponsorShip Pass',
+        SponsorContent:
+            'PyCon TW events including PyCon APAC 2022 main conference, Sprints, Job Fair, Open Space, PyNight, ' +
+            'PyCast, {br} local Python community events, as well as Young inspirers.' +
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:26]: nit: consider adding the link to the event pages

File: i18n/registration/tickets.i18n.js [hunk 6]
```diff
@@ -19,8 +19,15 @@ export default genI18nMessages({
             'with our Super Early Bird prices. ' +
             'The fee you spend will be put straight back into the community to ' +
             'support our diversity initiatives and financial aid programs. ' +
-            'If you need financial aid, please visit {fa}.{br}' +
-            'Limited edition conference swags Pyckage will be revealed soon.',
+            'If you need financial aid, please visit {fa}.{br}',
+        introSponsor: 'About SponsorShip Pass',
+        SponsorContent:
+            'PyCon TW events including PyCon APAC 2022 main conference, Sprints, Job Fair, Open Space, PyNight, ' +
```
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:25]:
```suggestion
            'Sponsors are what make PyCon TW possible. ' +
            'PyCon TW events this year includes PyCon APAC 2022 main conference, Sprints, Job Fair, Open Space, PyNight, ' +
```

File: i18n/registration/tickets.i18n.js (hunk 3 above)
Comment (mattwang44) [inline i18n/registration/tickets.i18n.js:30]:
```suggestion
            'Purchase the sponsorship ticket today and help us ensure PyCon TW keeps thriving!'
```

**Review by mattwang44 — COMMENTED** [review]
The grammar looks weird to me. Try to come up with a revision here but it's better to have a review from a native speaker.

**Comment by tyuchx** [comment]
先將個人贊助票的開賣連結開啟，等個人贊助票的簡介文案在 Review 過後再放上官網

CleanShot 2022-07-13 at 04 01 16

CleanShot 2022-07-13 at 04 01 16

**Pull request merged**